docs/source/optim.rst

```diff
@@ -270,16 +270,44 @@ SWA has been proposed in `Averaging Weights Leads to Wider Optima and Better Gen
 Constructing averaged models
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-`AveragedModel` class serves to compute the weights of the SWA model. You can create an
-averaged model by running:
+The `AveragedModel` class serves to compute the weights of the SWA or EMA model.
+
+You can create an SWA averaged model by running:
+
+>>> averaged_model = AveragedModel(model)
 
->>> swa_model = AveragedModel(model)
+EMA models are constructed by specifying the ``multi_avg_fn`` argument as follows:
 
-Here the model ``model`` can be an arbitrary :class:`torch.nn.Module` object. ``swa_model``
+>>> averaged_model = AveragedModel(model, multi_avg_fn=get_ema_multi_avg_fn(decay))
+
+
+Here the model ``model`` can be an arbitrary :class:`torch.nn.Module` object. ``averaged_model``
 will keep track of the running averages of the parameters of the ``model``. To update these
 averages, you can use the :func:`update_parameters` function:
```

Comment:
This is not the fault of this PR, but since we are here, I might as well point it out: it is unclear whether it is intended for the user to call update_parameters and when. From the usage below, it looks like it should be called after the optimizer.step() after a specified step number (swa_start) and it would be good to clarify this usage.

Reply:
Added a sentence to clarify